pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -31,16 +31,16 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
-// containerGC is the manager of garbage collection.
-type containerGC struct {
+// ContainerGC is the manager of garbage collection.
+type ContainerGC struct {
```

Comment:
Is this type being exported anywhere? I'm curious why golint thinks this is an error if it was working correctly without the capitalization.

Reply:
it is privated. golint thinks the exported func return an private is confused.
' func NewContainerGC(client internalapi.RuntimeService, podStateProvider podStateProvider, manager *kubeGenericRuntimeManager)  *ContainerGC '
keep the type private, and also change the func to private. WDYT? @shubheksha

Reply:
Is the function `NewContainerGC(..)` being called from elsewhere? If not, I think it makes sense to not make this public.

Reply:
fixed. PTAL, Thanks! @shubheksha